server/testsupport/aValidEducationDtoTestDataBuilder.ts

```diff
@@ -0,0 +1,30 @@
+import type { EducationResponse } from 'educationAndWorkPlanApiClient'
+import EducationLevelValue from '../enums/educationLevelValue'
+import QualificationLevelValue from '../enums/qualificationLevelValue'
+
+const aValidEducationDto = (): EducationResponse => ({
+  createdBy: 'asmith_gen',
+  createdByDisplayName: 'Alex Smith',
+  createdAt: '2023-06-19T09:39:44Z',
```

Comment:
If we can change this function to return `EducationDto` you will probably find all these date properties (`createdAt` and `updatedAt`) will need to change to set a real date, rather than a string ... which in turn might mean your mapper will need some love .... 🤔  (shouldnt be too hard though)

Reply:
Refactored.